services/bots/src/discord/commands/line_count_enforcer.ts

```diff
@@ -0,0 +1,20 @@
+import { On } from '@discord-nestjs/core';
+import { Message, AttachmentBuilder } from 'discord.js';
+
+const MAX_LINE_LENGTH = 17;
+
+export class LineCountEnforcer {
+  @On('messageCreate')
+  async handler(message: Message): Promise<void> {
+    if (message.content.split('\n').length > MAX_LINE_LENGTH) {
+      const attachment = new AttachmentBuilder(Buffer.from(message.content, 'utf-8'), {
+        name: 'message',
```

Comment:
```suggestion
        name: 'message.txt',
```
This allows preview and not only download